mono/metadata/marshal.c

```diff
@@ -7793,6 +7793,8 @@ mono_marshal_get_native_wrapper (MonoMethod *method, gboolean check_exceptions,
 		int thread_info_var = -1, stack_mark_var = -1, error_var = -1;
 		MonoMethodSignature *call_sig = csig;
 		gboolean uses_handles = FALSE;
+		int num_outarg_handles = 0;
```

Comment:
This should be a boolean, we don't use the number of outarg handles for anything